addons/l10n_ph/data/account_fiscal_position_tax_template.xml

```diff
@@ -0,0 +1,160 @@
+        <field name="tax_src_id" ref="tax_template_purchase_non_vat" />
+        <field name="tax_dest_id" ref="tax_template_purchase_vat_exempt" />
+        <field name="position_id" ref="fiscal_position_template_vat_exempt" />
+    </record>
+    <record
+        id="fiscal_position_tax_template_purchase_vat_exempt_zero_rated"
+        model="account.fiscal.position.tax.template"
+    >
+        <field name="tax_src_id" ref="tax_template_purchase_zero_rated" />
+        <field name="tax_dest_id" ref="tax_template_purchase_vat_exempt" />
+        <field name="position_id" ref="fiscal_position_template_vat_exempt" />
+    </record>
+    <record
+        id="fiscal_position_tax_template_sale_zero_rated_vat_registered"
+        model="account.fiscal.position.tax.template"
+    >
+        <field name="tax_src_id" ref="tax_template_sale_vat_12" />
+        <field name="tax_dest_id" ref="tax_template_sale_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+    <record id="fiscal_position_tax_template_sale_zero_rated_non_vat" model="account.fiscal.position.tax.template">
+        <field name="tax_src_id" ref="tax_template_sale_non_vat" />
+        <field name="tax_dest_id" ref="tax_template_sale_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+    <record id="fiscal_position_tax_template_sale_zero_rated_vat_exempt" model="account.fiscal.position.tax.template">
+        <field name="tax_src_id" ref="tax_template_sale_vat_exempt" />
+        <field name="tax_dest_id" ref="tax_template_sale_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+    <record
+        id="fiscal_position_tax_template_purchase_zero_rated_vat_registered"
+        model="account.fiscal.position.tax.template"
+    >
+        <field name="tax_src_id" ref="tax_template_purchase_vat_12" />
+        <field name="tax_dest_id" ref="tax_template_purchase_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+    <record id="fiscal_position_tax_template_purchase_zero_rated_non_vat" model="account.fiscal.position.tax.template">
+        <field name="tax_src_id" ref="tax_template_purchase_non_vat" />
+        <field name="tax_dest_id" ref="tax_template_purchase_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+    <record
+        id="fiscal_position_tax_template_purchase_zero_rated_vat_exempt"
+        model="account.fiscal.position.tax.template"
+    >
+        <field name="tax_src_id" ref="tax_template_purchase_vat_exempt" />
+        <field name="tax_dest_id" ref="tax_template_purchase_zero_rated" />
+        <field name="position_id" ref="fiscal_position_template_zero_rated" />
+    </record>
+</odoo>
```

Comment:
no purchase taxes in the fiscal positions at all?

Reply:
Yes we don't need it because the all our clients will edit them manually in the database. There is no standard fiscal position taxes in ph. For the vat_exempt, they will know that from the beginning the clients are vat_exempt or not.